### PR TITLE
TravisCI and PyUp Bot integration

### PR DESCRIPTION
**Pull request by avinashsai**
Hi @ibrahimsharaf,
I ran some tests with travis CI and checks were successful.
After the tests these are the results
Testing accuracy and Testing F1-Score are more than 80%.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -66,7 +71,7 @@ def train_doc2vec(corpus):
     # 10 epochs take around 10 minutes on my machine (i7), if you have more time/computational power make it 20
     for epoch in range(10):
         logging.info('Training iteration #{0}'.format(epoch))
-        d2v.train(corpus, total_examples=d2v.corpus_count, epochs=d2v.iter)
+        d2v.train(corpus, total_examples=d2v.corpus_count, epochs=d2v.epochs)
```
Comment (ibrahimsharaf) [inline model.py]: - Shall we reverse the changes that won't break the script (except of this line, because of gensim update). and leave the other modifications to a separate PR after merging this one? 
- To clarify, let's leave this PR for Travis and pyup integrations, and needed gesnim syntax modifications.

**Comment by ibrahimsharaf** [comment]
Great work @avinashsai, I left a couple of review comments for you.

Another question, where is the Travis build log? shouldn't it start showing here (in each PR)?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+ language: python
+ python:
+    - "3.5"
+ install:
+  - sudo rm -f /etc/boto.cfg
```
[the remark this answers is not in the document]
Comment (avinashsai) [reply .travis.yml]: I added that based on this https://github.com/travis-ci/travis-ci/issues/7940 issue. I was getting "No module named google_compute_engine" error before adding that. Adding that line solved the error.

File: model.py (hunk 1 above)
Comment (avinashsai) [reply model.py]: Cool then I will revert all the changes except for requirements.txt

File: .travis.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ibrahimsharaf) [reply .travis.yml]: Nice catch.

**Comment by ibrahimsharaf** [comment]
I am not really sure, can you send me the build log link?

File: model.py (hunk 1 above)
Comment (ibrahimsharaf) [reply model.py]: except of this line, because using `iter` instead of `epochs` with the new gensim version will fail, right?

Comment (avinashsai) [reply model.py]: Yes, it will throw a deprecation warning.

**Comment by avinashsai** [comment]
https://api.travis-ci.org/v3/job/480541470/log.txt This is the log.

**Comment by ibrahimsharaf** [comment]
Looking nice, but I think that we should add a new `test_model.py` script in this PR, which runs a simple test which trains the model, and gets the F1 score and ensures that it is >70%, that way TravisCI will tell us whether the test is passing or failing (instead of just running it), what do you think?

**Comment by avinashsai** [comment]
Cool then I will revert all changes except for that new syntax change (epochs=d2v.epochs instead of epochs=d2v.iter)

**Comment by avinashsai** [comment]
I will close this PR and will open a new PR with a clean title.
